### DISCUSSION TITLE
Split level 1 section menu into top level nav bar

### DISCUSSION THREAD

**Discussion by danielmaharry-okta** [google/docsy#1912] [Ideas]
Hi there, 
Our nav tree goes six levels deep. Would it be possible to have it so the top nav bar has all the level 1 nav headings and the left nav bar shows all the items that sit under whichever level 1 nav item is currently selected in the top bar? This would be _really_ useful.
Thanks

**Comment by fekete-robert** [comment]
Currently that's not possible AFAIK, and it probably would need some heavy customization to make it work. Wouldn't be enough to increase the default width of the sidebar?

**Reply by danielmaharry-okta** [reply]
To a degree. I ask as our current solution does this split between the two navbars already and am looking for a replacement. What with the indent for a sixth level menu item and the length of the title, it's not a great user experience.
